### DISCUSSION TITLE
iOS App - do not show local photos

### DISCUSSION THREAD

**Discussion by langepil** [immich-app/immich#4477] [Q&A]
Is it possible to not show the local images in the iOS app. I would like to build an independent media library that has nothing or little to do with the local iOS media library. Accordingly, I would not like to see the local pictures in the app, but still have the possibility to upload some of them.

**Comment by ddshd** [comment]
You can limit the photo gallery access on iOS to certain photos, or no photos as a workaround for now

**Reply by langepil** [reply]
I'm aware of that, just unfortunately it's not practical for me. Firstly, when I open the app, I always get a warning message and secondly, it is quite cumbersome to upload photos. I would have to go to the settings and add the new photos every time. 
A switch in the settings would be relatively easy to implement. Is that possibly planned?
